Kibol-Alert/Services/JwtHelper.cs

```diff
@@ -17,15 +17,12 @@ public class JwtHelper : IJwtHelper
         private readonly Kibol_AlertContext _context;
         private readonly AppSettings _appSettings;
 
-
         public JwtHelper(IOptions<AppSettings> appSettings, Kibol_AlertContext context)
         {
             _appSettings = appSettings.Value;
             _context = context;
-
         }
 
-
         JwtToken IJwtHelper.GenerateJwtToken(string username)
```

Comment:
delete here prefix "IJwtHelper." because you dont need it here

Reply:
Fixed